mainapp/templates/account/login.html

```diff
@@ -44,7 +46,14 @@ <h1>{% trans "Sign In" %}</h1>
         <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
         {% endif %}
         <a class="button secondaryAction" href="{% url 'account_reset_password' %}">{% trans "Forgot Password?" %}</a>
-        <button class="btn btn-primary primaryAction" type="submit">{% trans "Sign In" %}</button>
+        <div class="row">
+        <div class="col-lg">
+          <button class="btn btn-primary primaryAction" type="submit">{% trans "Sign In" %}</button>
+        </div>
+        <div class=""col-lg">
+        <button type="button" class="btn btn-primary" onclick="history.back()">Back</button></a>
+        </div>
+        </div>
```

Comment:
is there a reason why the `/accounts/login.html` is being updated in your PR?

Reply:
I didn't update or maybe I updated last time. Now what should I do ?